shared/src/main/scala/tastyquery/ast/Symbols.scala

```diff
@@ -64,6 +69,18 @@ object Symbols {
       if isFlagsInitialized then myFlags
       else throw IllegalStateException(s"flags of $this have not been initialized")
 
+    final def is(flag: Flag): Boolean =
```

Comment:
seems like (ideally if not already possible) `export` should be able to cover this, even if its adapting an extension method on an opaque type to a normal method on the outer class.

Reply:
(other than flags being a def)

Reply:
`flags` is a `def` so it's not even stable. We can't export from it, regardless of whether it is extension methods or not.